master/buildbot/reporters/stash.py

```diff
@@ -82,3 +90,52 @@ def send(self, build):
                 content = yield response.content()
                 log.error("{code}: Unable to send Stash status: {content}",
                           code=response.code, content=content)
+
+
+class StashPRCommentPush(http.HttpStatusPushBase):
+    name = "StashPRCommentPush"
+
+    @defer.inlineCallbacks
+    def reconfigService(self, base_url, user, password, text=None, verbose=False, **kwargs):
+        yield http.HttpStatusPushBase.reconfigService(self, wantProperties=True,
+                                                      **kwargs)
+        self.text = text or Interpolate('Builder: %(prop:buildername)s Status: %(prop:statustext)s')
```

Comment:
yes, I've seen it everywhere (this class was based on StashStatusPush so it may be akin) . It seems an easy way to set a default value, do you think it's hard to read?
I've also changed StashStatusPush to use got_revision if revision is not set (meaning the latest revision), or skip the change if none are present.